### PR TITLE
Update extension structure

### PR DESCRIPTION
**Pull request by WilliamCunhaCardoso**
Related to: https://github.com/jonataslaw/getx/issues/389

### CODE REVIEW

File: lib/src/extensions/widget_extensions.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/widgets.dart';
+
+extension WidgetPaddingX on Widget {
```
Comment (jonataslaw) [inline lib/src/extensions/widget_extensions.dart:3]: Wow, it's cool!
I was thinking about doing this this weekend, I'm glad you thought about it.

Comment (WilliamCunhaCardoso) [reply lib/src/extensions/widget_extensions.dart:3]: Thanks!

**Comment by WilliamCunhaCardoso** [comment]
@jonataslaw I haven't finished all the tests but I prepared the structure to it. Actually I intend to finish it but I also want to leave this structure available to new contributions. This is the reason I removed WIP.

Please give me some feedback

**Comment by jonataslaw** [comment]
It looks good to me.
As there were no tests yet, so there is no chance of breaking with the current insert, if you want I can merge, and you can add the other tests later.

**Comment by jonataslaw** [comment]
hmm, lib/src/context_extensions/extensions.dart is conflicting

**Comment by WilliamCunhaCardoso** [comment]
> It looks good to me.
> As there were no tests yet, so there is no chance of breaking with the current insert, if you want I can merge, and you can add the other tests later.

Nice!. I would appreciate it

**Comment by WilliamCunhaCardoso** [comment]
> hmm, lib/src/context_extensions/extensions.dart is conflicting

I cannot see what is conflicting. I have no option to resolve it. 
But, I guess it is because I moved this file to an extensions directory.

**Comment by Nipodemos** [comment]
is problably because of my PR that added documentation, i'll try to mess here with gitkraken and see if it helps

**Comment by Nipodemos** [comment]
ok i think i've messed up 😓 , let me try to fix

**Comment by Nipodemos** [comment]
sorry @WilliamCunhaCardoso i had to force-push to your branch to fix the conflicts, so it means that your fork might get some problems. I'm still not too good at github when is needed to make those complicated things

**Comment by WilliamCunhaCardoso** [comment]
No problem, those things just happens xD. Looks like there is no problem!

**Comment by jonataslaw** [comment]
I tried to resolve the conflicts, but it seems that Git did not identify the change in the folder structure on the master, and it is as if the files had been deleted.
I don't know how to proceed now, as the context_extensions are now at https://github.com/jonataslaw/getx/tree/master/lib/src/utils/context_extensions

And the dynamic_utils file reviews removed methods, and StringUtils added the cpf and cnpj methods.

Would you be able to enjoy this PR without having to completely redo it?

**Comment by Nipodemos** [comment]
i  think is more pratical to redo the PR, i've messed with this once and it was not nice

**Comment by Nipodemos** [comment]
yeah, i think i've done what is need to be merge
however, i don't know how to test this... @jonataslaw can you test if this code is running? My VSCode was giving a very strange error but to be honest i don't even know how to run  this

**Comment by WilliamCunhaCardoso** [comment]
Sorry being late. I saw it was talking about test. But I made test for it, didn't I?

**Comment by Nipodemos** [comment]
it is because i had to basically redo your PR, move the files around, and i don't know how to check if it was everything working and imported
but jonatas approved, so it's ok 😅

**Pull request merged**